Examples/Geometry/VR/index.js

```diff
@@ -67,7 +66,7 @@ mapper.setInputConnection(filter.getOutputPort());
 
 const actor = vtkActor.newInstance();
 actor.setMapper(mapper);
-actor.setPosition(20.0, 0.0, 0.0);
+actor.setPosition(-1.5, 0.0, 0.0);
```

Comment:
Same comment as above. The reset code should work so that it doesn't matter where the code is or what size it is, it should default to a nice view and interaction in VR.

Reply:
Need to explore view resetting with the [WebXR gamepads module](https://github.com/immersive-web/webxr-gamepads-module), at the moment we're purely using the view we get from the XR session.

Reply:
Resolved above, was skipping the world-to-physical transformation with `computeViewParametersFromViewMatrix`. Gamepad not required for the VR example in this first pass.